java/mykola-melnyk/src/com/zoolatech/lecture3/tasks/_2/UserAccountValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.zoolatech.lecture3.tasks._2;
+
+import com.zoolatech.lecture3.tasks._2.validators.*;
```

Comment:
please don't use `*` import

Reply:
Created this:
import com.zoolatech.lecture3.tasks._2.validators.BiggerThan;
import com.zoolatech.lecture3.tasks._2.validators.StringNotEmpty;
import com.zoolatech.lecture3.tasks._2.validators.ValidEmail;
import com.zoolatech.lecture3.tasks._2.validators.Validator;
import com.zoolatech.lecture3.tasks._2.validators.ValidPhoneNumber;
import com.zoolatech.lecture3.tasks._2.validators.LessThan;
import com.zoolatech.lecture3.tasks._2.validators.NumberRangeXY;

But why do we need to do this instead of using * ?